### PR TITLE
Add cvss types

### PR DESCRIPTION
**Pull request by himynameisdave**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [X] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [X] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [X] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [X] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [X] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [X] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/cvss/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+      "module": "commonjs",
+      "lib": [
+          "es5"
+      ],
+      "allowSyntheticDefaultImports": true,
+      "esModuleInterop": true,
+      "noImplicitAny": true,
```
Comment (peterblazejewicz) [inline types/cvss/tsconfig.json:9]:
```suggestion
      "noImplicitAny": true,
```
please do not modify default config, unless there is a valid reason:
https://github.com/DefinitelyTyped/DefinitelyTyped#esmoduleinteropallowsyntheticdefaultimports

Comment (himynameisdave) [reply types/cvss/tsconfig.json:9]: Sorry, I copied + pasted this from a time I had contributed previously

File: types/cvss/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+      "module": "commonjs",
+      "lib": [
+          "es5"
```
Comment (peterblazejewicz) [inline types/cvss/tsconfig.json:5]:
```suggestion
          "es6"
```
it's internal across DT packages, not really used when consuming the package. Do you have specific requirement here?

Comment (himynameisdave) [reply types/cvss/tsconfig.json:5]: nah, again just copy-pasting from a previous contribution.

**Review by peterblazejewicz — APPROVED** [review]
LGMT!
@himynameisdave thnaks!

**Comment by himynameisdave** [comment]
Ready to merge

**Pull request merged**